typings/audit.d.ts

```diff
@@ -7,7 +7,7 @@
 declare global {
   module LH.Audit {
     export interface Context {
-      options: Object; // audit options
+      options: Record<string, any>; // audit options
```

Comment:
what's the difference between `Record<K, V>` and `Object<K, V>`

Reply:
> what's the difference between Record<K, V> and Object<K, V>

`Object<K, V>` only works in jsdoc. It's not parameterized in typescript code

Reply:
...we should probably come up with some sort of style rule for this vs `{[p: string]: any}` vs `Object<>` on the jsdoc side